libyul/AsmAnalysis.cpp

```diff
@@ -279,10 +284,20 @@ vector<YulString> AsmAnalyzer::operator()(FunctionCall const& _funCall)
 		}
 	}))
 	{
-		if (!warnOnInstructions(_funCall.functionName.name.str(), _funCall.functionName.location))
-			declarationError(_funCall.functionName.location, "Function not found.");
+		runWarnOnInstructions = true;
+		reportDeclarationError = true;
 		m_success = false;
 	}
+
+	bool const warnOnInstructionsFailed =
+		!runWarnOnInstructions ||
+		!warnOnInstructions(_funCall.functionName.name.str(), _funCall.functionName.location);
+
```

Comment:
I find this part a bit hard to read.

Reply:
with a new git rebase I had to rewrite it nevertheless.